features/admin.users.v1/components/wizard/steps/add-user-basic.tsx

```diff
@@ -1719,6 +1677,43 @@ export const AddUserUpdated: React.FunctionComponent<AddUserProps> = (
             );
         }
 
+        // If SMS OTP is enabled and ask password option is EMAIL, show mandatory mobile field
+        if (askPasswordVerificationOption === VerificationOption.SMS_OTP &&
+            askPasswordOption === AskPasswordOptionTypes.EMAIL) {
```

Comment:
We may refactor this constant later. It denotes whether offline or online invite. As previous online invite was email, the constant has been named as that.